docs/runtimes/configuring-job-resources.md

```diff
@@ -318,29 +318,63 @@ Modify the priority for an ML function by pressing **ML functions**, then **<img
 of the function, **Edit** | **Resources** | **Pods Priority** drop-down list.
 
 ## Node selection
+
+```{admonition} Note
+Requires Nuclio v1.13.5 or higher.
+```
 Node selection can be used to specify where to run workloads (e.g. specific node groups, instance types, etc.). This is a more advanced 
-parameter mainly used in production deployments to isolate platform services from workloads. 
+parameter mainly used in production deployments to isolate platform services from workloads. Node selection can be specified in the MLRun 
+and in the Nuclio service level, at the function level, and at the project level. Configurations defined at the function level take 
+precedence over those at the project level. Configurations set at either the project or function level (or both) take precedence over 
+those at the service level: if any configuration is specified at the project or function level (or both), the service level 
+configuration is not considered.   
+
```

Comment:
This actually might be changed soon, so that the service level configuration will take effect, with precedence: function -> project -> service.
Will update once the PR is merged, but I guess it can already be adjusted accordingly (need to adjust examples below as well, lmk if you have any questions🙏🏻)

Reply:
@yaelgen will it be changed for v1.7? or after?

Reply:
Yes, for 1.7.0